### DISCUSSION TITLE
Access id of the AccordionItem

### DISCUSSION THREAD

**Discussion by pdhruv93** [chakra-ui/chakra-ui#8379] [Q&A] [answered]
How to access the `id` prop of the AccordionItem inside onChange when the item is expanded. Right now only index is accessible.
Also how to get the id inside AccordionButton, AccordionPanel

**Comment by segunadebayo** [comment, marked as answer]
v3 Accordion uses `value` strings, not index. Read `details.value` in `onValueChange`.
